src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,78 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
 public class SalaryInfo {
+    private static final int dateIndex = 0;
```

Comment:
```suggestion
    private static final int DATE_INDEX = 0;
```
Use the correct constant format. Fix it for other constants